src/server/index.ts

```diff
@@ -0,0 +1,68 @@
+import express from "express"; // importing
+import axios from "axios";
+import "./config";
+import geohash from "ngeohash";
+
+const app = express(); // creating an app, which is the server
+const PORT = 8000; // which port the server will run on
+
+const postCode = "SW10 0EF";
+
+const API_KEY = process.env.TICKET_MASTER_API_KEY;
+const city = "London";
+
+app.get("/", (req, res) => res.send("Express + TypeScript Server")); //defining your handler
```

Comment:
Suggestion: lots of API developers add status check routes that they can call to check quickly if the API is up or down. I would suggest editing this route to something like `/ping` and return `pong` (or something similar) instead of the base `/` which is not specific enough for an API.